app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -48,14 +49,13 @@ def verified_after_consent?(last_estimated_consent)
     verification_timestamp.present? && last_estimated_consent < verification_timestamp
   end
 
-  def sp_session_identity
-    @sp_session_identity =
-      current_user&.identities&.find_by(service_provider: sp_session[:issuer])
+  def find_sp_session_identity
+    current_user&.identities&.find_by(service_provider: sp_session[:issuer])
```

Comment:
what if we left this in but made the memoization nil-safe?

```suggestion
    return @sp_session_identity if defined?(@sp_session_identity)
    @sp_session_identity =
      current_user&.identities&.find_by(service_provider: sp_session[:issuer])
```

Reply:
blah, rubocop got mad because the method doesn't match the memoized variable, but I did realize I can change the `@sp_session_identity` assignment towards the top since there's no need to make it an instance variable

Reply:
(we could keep the old method name too? but at the end of the day, same result)